package.json

```diff
@@ -22,8 +22,8 @@
     "coverage": "nyc npm run test && npm run coverage:report",
     "reports:show": "open ./reports/*",
 
-    "start:dev": "nodemon --watch main.js --watch app --ignore app/public",
-    "start:prod": "node main.js"
+    "start:dev": "nodemon --watch main.js --watch app --ignore app/public -- noid",
+    "start:prod": "node main.js noid"
   },
   "dependencies": {
     "@aliceo2/web-ui": "^1.18.2",
```

Comment:
(out of topic comment) Latest version of the AliceO2/WebUI library is now : `1.23.1`
It might be good to update: https://www.npmjs.com/package/@aliceo2/web-ui